arkouda/groupbyclass.py

```diff
@@ -1868,14 +1868,14 @@ def broadcast(
 
     Examples
     --------
+    >>>
     # Define a sparse matrix with 3 rows and 7 nonzeros
     >>> row_starts = ak.array([0, 2, 5])
     >>> nnz = 7
     # Broadcast the row number to each nonzero element
     >>> row_number = ak.arange(3)
     >>> ak.broadcast(row_starts, row_number, nnz)
     array([0 0 1 1 1 2 2])
-
     # If the original nonzeros were in reverse order...
```

Comment:
we need the same edit for the class level `broadcast` doc string

Reply:
Updated